cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -239,8 +238,7 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 		glog.Warningf("Unsuccessful parsing of service CIDR %v: %v", s.ServiceCIDR, err)
 	}
 	nodeController := nodecontroller.NewNodeController(cloud, clientset.NewForConfigOrDie(restclient.AddUserAgent(kubeconfig, "node-controller")),
-		s.PodEvictionTimeout.Duration, flowcontrol.NewTokenBucketRateLimiter(s.DeletingPodsQps, int(s.DeletingPodsBurst)),
-		flowcontrol.NewTokenBucketRateLimiter(s.DeletingPodsQps, int(s.DeletingPodsBurst)),
+		s.PodEvictionTimeout.Duration, s.DeletingPodsQps, int(s.DeletingPodsBurst), s.DeletingPodsQps, int(s.DeletingPodsBurst),
```

Comment:
Do you have any usecase on your mind of having different deletionEviction & terminationEvictions rate-limitters? I mean with different parameters.
Currently, there is no way to have different values for them, so maybe they should have always the same values, and then we can pass those values once instead of twice.